shacl/shacl-shacl.ttl

```diff
@@ -182,6 +184,7 @@ shsh:ShapeShape
 		sh:path sh:maxCount ;
 		sh:datatype xsd:integer ;       # maxCount-datatype
 		sh:maxCount 1 ;                 # maxCount-maxCount
+		sh:minInclusive 0 ;              # maxCount-minInclusive
```

Comment:
I was about to note that this seemed redundant with some encoding, but, no, the datatype is `xsd:integer`, so we don't actually have a floor encoded.  I'm fine with this addition, especially since the alternative of using a `xsd:nonNegativeInteger` datatype would induce a lot of extra keyboard work for shape authors.

Reply:
Right, the implicit design assumption in SHACL is that we only ever use xsd:integer and xsd:decimal for numbers. Anything else can be IMHO better expressed with sh:minInclusive etc.